duckduckgo/iOS
# new background tab

Task/Issue URL: https://app.asana.com/0/392891325557410/770186107328906
Tech Design URL:
CC:

**Description**:

Open tab in new background.

Note, waiting on final animation, but that should just be a change to the animation JSON.

**Steps to test this PR**:

1. Long press a link and select "open in background tab"

  * tab switcher button animates
  * number of tabs increments
  * tab switcher button shows correct number of tabs and unread tab dot

2. Open tab switcher

  * current tab has blue border
  * other tabs are slightly transparent
  * unread tabs have a blue dot
  * tab title might be the domain name with `www.` prefix stripped if the tab hasn't had chance to load yet

3. Select tab

 * Tab opens
 * Number of tabs remains the same
 * If there are no unread tabs the dot is removed
 * Open tab switcher -> selected tab now has blue border and others are slightly transparent

4. Confirm that open in background tab pixel has been fired (`mlp_b`)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
